spec/integration/geo_config_spec.rb

```diff
@@ -69,13 +69,57 @@ def stub_mods_request(druid, body)
       expect(result['solr_year_i']).to eq [1603]
     end
   end
+  context 'image map book content without dc:type' do
+    let(:druid) { 'ny179kk3075' }
+    before do
+      stub_purl_request(druid, File.read(file_fixture("#{druid}.xml").to_s))
+    end
+    it 'includes the layer_geom_type_s' do
+      expect(result).to include 'layer_geom_type_s' => ['Image']
+    end
+
+  end
   context 'for geo content' do
     let(:druid) { 'vv853br8653' }
     before do
       stub_purl_request(druid, File.read(file_fixture("#{druid}.xml").to_s))
     end
-    it 'skips record' do
-      expect(result).to be_nil
```

Comment:
Matching to https://raw.githubusercontent.com/OpenGeoMetadata/edu.stanford.purl/38e2095e99f772c34c7cf32818a58c1216be4045/vv/853/br/8653/geoblacklight.json